mobile/src/main/java/org/openhab/habdroid/ui/AbstractItemPickerActivity.kt

```diff
@@ -25,43 +24,45 @@ import android.widget.EditText
 import android.widget.TextView
 import androidx.appcompat.widget.SearchView
 import androidx.appcompat.widget.Toolbar
-import androidx.core.content.edit
-import androidx.core.os.bundleOf
 import androidx.core.view.isVisible
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
 import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
 import kotlinx.coroutines.Job
 import kotlinx.coroutines.launch
-
 import org.json.JSONArray
 import org.json.JSONException
 import org.openhab.habdroid.R
 import org.openhab.habdroid.core.connection.ConnectionFactory
 import org.openhab.habdroid.model.Item
 import org.openhab.habdroid.model.toItem
 import org.openhab.habdroid.ui.widget.DividerItemDecoration
-import org.openhab.habdroid.util.*
-
-class ItemPickerActivity : AbstractBaseActivity(), SwipeRefreshLayout.OnRefreshListener,
+import org.openhab.habdroid.util.HttpClient
+import org.openhab.habdroid.util.SuggestedCommandsFactory
+import org.openhab.habdroid.util.TaskerIntent
+import org.openhab.habdroid.util.getPrefs
+import org.openhab.habdroid.util.isTaskerPluginEnabled
```

Comment:
The Tasker specific stuff at the end of onCreate should be moved to the derived class as well, shouldn't it?

Reply:
Yes, forgot to remove them.